openid-4-verifiable-presentations-1_0.md

```diff
@@ -2810,6 +2825,91 @@ The following is a non-normative example of the unsecured payload of the Key Bin
+* `iss` Claim MUST represent the Credential Issuer.
+* `status` Claim MUST represent the information to obtain the status of the Credential.
+* `sub` Claim MUST represent the subject identifier of the Credential.
+
+IETF SD-JWT VC is extended with the following claims:
+
+* `ld`: OPTIONAL. Contains a JSON-LD [@!JSON-LD] object in compact form, e.g., [@?VC_DATA].
+
+#### Processing
+
+The following outlines a suggested non-normative set of processing steps for SD-JWT VCDM:
+
+##### Step 1: SD-JWT VC Processing
+
+- A receiver (holder or verifier) of an SD-JWT VCDM applies the processing rules outlined in Section 4 of [@!I-D.ietf-oauth-sd-jwt-vc], including verifying signatures, validity periods, status information, etc.
+- If the `vct` value is associated with any SD-JWT VC Type Metadata, schema validation of the entire SD-JWT VCDM is performed, including the nested `ld` claim.
+- Additionally, trust framework rules are applied, such as ensuring the Credential Issuer is authorized to issue SD-JWT VCDMs for the specified `vct` value.
+
+##### Step 2: Business Logic Processing
+
+- Once the SD-JWT VC is verified and trusted by the SD-JWT VC processor, and if the `ld` claim is present, the receiver extracts the JSON-LD object from the `ld` claim and uses this for the business logic object. If the `ld` claim is not present, the entire SD-JWT VC is considered to represent the business logic object.
+- The business logic object is then passed on for further use case-specific processing and validation. The business logic assumes that all security-critical functions (e.g., signature verification, trusted issuer) have already been performed during the previous step. Additional schema validation is applied if provided in the `ld` claim, e.g., to support SHACL schemas. Note that while a `vct` claim is required, SD-JWT VC type metadata resolution and related schema validation is optional in certain cases.
+
+#### Examples
+
+The following is a non-normative example of an unsecured payload of an SD-JWT VCDM (i.e., before applying the modifications to enable selective disclosure and before adding validity claims). This example is derived from the example of an unsecured payload in Section 3.3 of [@!I-D.ietf-oauth-sd-jwt-vc]:
+
+```json
+{
+   "vct": "https://credentials.example.com/identity_credential",
+   "ld":{
+      "@context":[
+         "https://www.w3.org/ns/credentials/v2",
+         "https://w3id.org/citizenship/v3",
+      ],
+      "type":[
+         "VerifiableCredential",
+         "PermanentResidentCard"
+      ],
+      "credentialSubject":{
+         "givenName": "John",
+         "familyName": "Doe",
+         "birthDate": "1978-07-17"
+      }
+   }
+}
+```
+
+The following is a non-normative example of the SD-JWT payload after enabling selective disclosure:
+```json
+{
+  "_sd": [
+    "vwUhdFvpylx9Sqi2YNBV1dfVK6lCbhXvkH0nThfKFT0"
```

Comment:
so the whole `ld` claim is made selectively disclosable? and nothing else that can be seen here anyway. what is this example showing?

Reply:
Resolved in the new example